test/psxy/vector2d_mods.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
To make the code validator happy